src/vertexcodec.cpp

```diff
@@ -1201,7 +1206,9 @@ int meshopt_decodeVertexBuffer(void* destination, size_t vertex_count, size_t ve
 
 	const unsigned char* (*decode)(const unsigned char*, const unsigned char*, unsigned char*, size_t, size_t, unsigned char[256]) = 0;
 
-#if defined(SIMD_SSE) && defined(SIMD_FALLBACK)
+#if defined(MESHOPTIMIZER_NO_SIMD)
```

Comment:
Is this block still necessary? It seems that if no SIMD_ defines are defined, this should correctly use scalar decoder.

Reply:
I didn't see it that clearly: the cpuid check below made me think that perhaps `decodeVertexBlockSimd` could be chosen, even if `MESHOPTIMIZER_NO_SIMD` is set. I can remove this if you are convinced, but perhaps we should add some documentation for the uninitiated (like me)?

Reply:
Well, the cpuid check is only performed if SIMD_SSE is defined - if SIMD_SSE isn't defined no SIMD code is compiled (or executed).

Reply:
Ah, good point! I wasn't looking very carefully and for some reason I thought it was `SIMD_SSE || SIMD_FALLBACK`. I removed this part and re-pushed.